src/components/ReceiveReviewComponent.jsx

```diff
@@ -79,28 +79,43 @@ class ReceiveReviewComponent extends Component {
                   <Typography className={classes.reviewSubtitle} align='left'>
                     Amount
                   </Typography>
-                  <Typography className={classes.reviewContentAmount} align='left' id='transferAmount'>
-                    {transferAmount} {getCryptoSymbol(cryptoType)}
-                  </Typography>
+                  <Grid container direction='column'>
+                    <Typography className={classes.reviewContentAmount} align='left' id='transferAmount'>
+                      {transferAmount} {getCryptoSymbol(cryptoType)}
+                    </Typography>
+                    <Typography className={classes.reviewContentCurrencyAmount} align='left' id='transferCurrencyAmount'>
+                      ≈ { toCurrencyAmount(transferAmount) }
+                    </Typography>
+                  </Grid>
                 </Grid>
                 <Grid item className={classes.reviewItem}>
                   <Typography className={classes.reviewSubtitle} align='left'>
                     Transaction Fee
                   </Typography>
                   {!actionsPending.getTxFee && txFee
-                    ? <Typography className={classes.reviewContent} align='left'>
-                      {txFee.costInStandardUnit} {getCryptoSymbol(getTxFeesCryptoType(cryptoType))}
-                    </Typography>
+                    ? <Grid container direction='column'>
+                      <Typography className={classes.reviewContentAmount} align='left'>
+                        {txFee.costInStandardUnit} {getCryptoSymbol(getTxFeesCryptoType(cryptoType))}
+                      </Typography>
+                      <Typography className={classes.reviewContentCurrencyAmount} align='left'>
+                        ≈ { toCurrencyAmount(txFee.costInStandardUnit) }
+                      </Typography>
+                    </Grid>
                     : <CircularProgress size={18} color='primary' />}
                 </Grid>
                 <Grid item>
                   <Typography className={classes.reviewSubtitle} align='left'>
                     You will receive*
                   </Typography>
                   {!actionsPending.getTxFee && txFee
-                    ? <Typography className={classes.reviewContent} align='left' id='receiveAmount'>
-                      {receiveAmount} {getCryptoSymbol(cryptoType)}
-                    </Typography>
+                    ? <Grid container direction='column'>
+                      <Typography className={classes.reviewContentAmount} align='left' id='receiveAmount'>
+                        {receiveAmount} {getCryptoSymbol(cryptoType)}
+                      </Typography>
+                      <Typography className={classes.reviewContentCurrencyAmount} align='left' id='receiveCurrencyAmount'>
+                        ≈ { toCurrencyAmount(receiveAmount) }
```

Comment:
move this function to the container, the component should not handle conversion.